message KRawImage{
	enum Colorspaces {
		YUYV = 1;
		OTHER  = 2;
	}
	enum Camera{
		TOP=0;
		BOTTOM=1;
		
	}
	

	required uint32 bytes_per_pix = 1 ;
	required uint32 width = 2 [default = 0];
	required uint32 height = 3 [default = 0];
	required Colorspaces colorspace = 4;
	required bytes image_rawdata = 5;
	required int32 exposure_us =6;
	required float luminance_scale=7;
	required Camera active_camera =8;
}


message KCalibrateCam{
	required int32 status =1;
	required int32 sleeptime =2 [default = 1500];
	required int32 exp= 3 [default =13];
	required float exposure_comp =4 [default=1];
}
